src/python/grpcio/grpc/_adapter/_types.py

```diff
@@ -165,7 +165,7 @@ class OpArgs(collections.namedtuple(
     message (bytes): Only valid if type == OpType.SEND_MESSAGE, else is None.
     status (Status): Only valid if type == OpType.SEND_STATUS_FROM_SERVER, else
       is None.
-    write_flags (int): a bit OR'ing of 0 or more OpWriteFlags values.
+    flags (int): a bit OR'ing of 0 or more OpWriteFlags values.
```

Comment:
While the flags element is available to all `grpc_op` types (whether the kind that writes *or not*) in the core, the specific flags exposed are only of the section titled "Write Flags" [here](https://github.com/grpc/grpc/blob/master/include/grpc/impl/codegen/grpc_types.h#L197) in core. I wouldn't mind the change, but at this point we're just trading naming for symmetry and there's no write answer.